Logger/FileLogger.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.IO;
+
+namespace Logger
+{
+   public class FileLogger : BaseLogger
+    {
+        private string _FilePath { get; }
+        public FileLogger(string filePath) { _FilePath = filePath; }
```

Comment:
```suggestion
        public FileLogger(string filePath) => _FilePath = filePath; 
```
If you want to keep this on a single line, use a lambda function, otherwise, the brackets should be multilined. 

However, for constructors, I would recommend always writing them this way:
```suggestion
        public FileLogger(string filePath) 
        { 
           _FilePath = filePath; 
        }
```